addons/skin.estuary/xml/VideoOSD.xml

```diff
@@ -15,7 +14,28 @@
 			<texturenofocus />
 			<onclick>Action(close)</onclick>
 		</control>
+		<control type="radiobutton" id="799">
```

Comment:
From description I'm guessing the Back button should only be shown when Touch mode is enabled? I'm getting it appearing with Touch mode disabled 

![image](https://user-images.githubusercontent.com/5781142/85923278-ceeaae80-b881-11ea-86fd-f1e3facac5f4.png)

Shouldn't be a `<visible>Skin.HasSetting(touchmode)</visible>` for this control.

It also looks like the similar controls for MusicOSD and Slideshow are also missing this visible condition.